core/database/user_limits_db.py

```diff
@@ -176,6 +208,47 @@ async def update_user_tier(
         except Exception as e:
             logger.error(f"Failed to update user tier: {e}")
             return False
+            
+    async def update_subscription_info(
```

Comment:
This is good, we have extra upgrade user tier function now that can be removed! From user_service, and from api!!